rpc_jobs/unit/artefact_publish.yml

```diff
@@ -6,12 +6,28 @@
       - build-discarder:
           num-to-keep: 30
     parameters:
+      - instance_params:
+          IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
```

Comment:
Where are these used? `common.globalWraps()` will allocate a long running slave, `common.standard_job_slave()` will create a container on that long running slave.

Reply:
I just grabbed them from the single_use_slave unit test because I thought they were necessary.